AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/mrecs/MrecAdActivity.kt

```diff
@@ -2,7 +2,7 @@ package com.applovin.enterprise.apps.demoapp.ads.max.mrecs
 
 import android.content.Intent
 import com.applovin.enterprise.apps.demoapp.ads.DemoMenuActivity
-import com.applovin.enterprise.apps.demoapp.ads.DemoMenuItem
+import com.applovin.enterprise.apps.demoapp.data.main.DemoMenuItem
```

Comment:
Seems like the Mrec activity actually is not working as expected, not sure what exactly is different for MERC and other activities, might look into it after:
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.applovin.enterprise.apps.demoapp/com.applovin.enterprise.apps.demoapp.ads.max.mrecs.ProgrammaticMrecAdActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void androidx.recyclerview.widget.RecyclerView.setHasFixedSize(boolean)' on a null object reference

Reply:
Feel free to fix :)